### DISCUSSION TITLE
Error while moving People from one to another category

### DISCUSSION THREAD

**Discussion by zickzackzuck** [nextcloud/recognize#824] [General]
Hi,

I just rented some serverspace and I'm quite new to Nextcloud. I've installed Recongnize and it works well. 
- I'm using Nextcloud 26.01 and Recognize 4.01

I wanted to move wrong recongized faces to another category, but it always give me back an error. If I look in the log-data from Nextcloud, there is no error code shown up. In the recognice Settings it says, I should look in the Error-logs.

Where do I find the logs for debugging? What is the error behind moving people from one category to another?

Thanks! robert

**Comment by Andreaux** [comment]
It sometimes happen to me as well. Seems like some lock or cache not updating right away. So far, if I reload the page and try again, it seems to work. Hope this helps.
